src/Map.jl

```diff
@@ -40,6 +57,24 @@ end
 #
 ################################################################################
 
+@doc raw"""
+    identity_map(R::D) where D <: AbstractAlgebra.Set
+
+Return an identity map on the domain $R$.
+
+# Examples
+```jldoctest; setup = :(using AbstractAlgebra)
+julia> R, t = ZZ[:t]
+(Univariate polynomial ring in t over integers, t)
+
+julia> f = identity_map(R)
```

Comment:
I could not yet figure out what causes the weirdness we saw before, but @thofma found a workaround: instead of `ZZ`, we use a polynomial ring, then it works.. No idea why, but let's move on for now.